src/bsoncxx/config/CMakeLists.txt

```diff
@@ -4,11 +4,6 @@ configure_file(
 )
 
 configure_file(
-    ${CMAKE_CURRENT_SOURCE_DIR}/private/config.hpp.in
```

Comment:
These actually do have a purpose. I don't think we should remove them. It is fine for them to remain empty for now. If we ever had any configure checks that resulted in HAVE macros that were purely internal, this is where they would go. So, I'd like to keep it.